packages/main/src/ColorPalettePopover.hbs

```diff
@@ -8,7 +8,7 @@
 >
 	<div slot="header" class="ui5-cp-header">
 		<ui5-title
-			class="ui5-cp-title"
+			level="H5"
```

Comment:
are we sure that the heading level should be h5? can we guarantee that the headings map will be correct?

Reply:
Yes, it's by design.